src/database/logger.ts

```diff
@@ -101,7 +101,7 @@ export class Logger {
         var rankName = rank.charAt(0).toUpperCase() + rank.slice(1);
         var embed: any = {
             "embed": {
-                "description": `${role ? 'Role ' + role.toString() : 'User' + user.toString()} was ${type ? 'removed' : 'added'} to ${rank} rank by ${mod.toString()}`,
+                "description": `${role ? 'Role ' + role.toString() : 'User' + user.toString()} was ${type ? 'removed' : 'added to'} ${rank} rank by ${mod.toString()}`,
```

Comment:
If a user or role gets removed it will say `User Username was removed rank by SomeMod`. There should also be a `from`

Reply:
Fixed!